src/commands/fact.ts

```diff
@@ -0,0 +1,29 @@
+exports.run = async (client, message) => { 
+    try {
+      const facts = ["77% of Cytrus-RE's code is just commands.", "Cytrus-RE was started in 2019.", "Odyssey346 is 2nd place on the Contributor page on our GitHub."]
```

Comment:
that third fact is a little biased and would easily become outdated

I think some stuff about our history would be better

also maybe something like copypasta's data file?

Reply:
will do.

Reply:
can you get me some history stuff?

Reply:
Hmmm...

"Cytrus-RE was originally based on a bot written by CelestialCrafter."

"At first, Cytrus-RE did not have a stable host."

uhhhh

"Cytrus-RE was initially written in JavaScript but has since switched to TypeScript"?